app/api/helpers/order.py

```diff
@@ -53,21 +53,20 @@ def create_pdf_tickets_for_holder(order):
     if order.status == 'completed':
         pdf = create_save_pdf(render_template('pdf/ticket_purchaser.html', order=order),
                               UPLOAD_PATHS['pdf']['ticket_attendee'],
-                              dir_path='/static/uploads/pdf/tickets/')
+                              dir_path='/static/uploads/pdf/tickets/', identifier=order.identifier, upload_dir='generated/tickets/')
         order.tickets_pdf_url = pdf
 
         for holder in order.ticket_holders:
             if (not holder.user) or holder.user.id != order.user_id:
                 # holder is not the order buyer.
                 pdf = create_save_pdf(render_template('pdf/ticket_attendee.html', order=order, holder=holder),
                                       UPLOAD_PATHS['pdf']['ticket_attendee'],
-                                      dir_path='/static/uploads/pdf/tickets/')
+                                      dir_path='/static/uploads/pdf/tickets/', identifier=order.identifier, upload_dir='generated/tickets/')
```

Comment:
What is `dir_path`

Reply:
`create_save_pdf` first creates the PDF temporarily in `dir_path`. After that, It moves the generated PDF to `upload_dir` and deletes the generated PDF in `dir_path`.

Reply:
OK